README.md

```diff
@@ -2,6 +2,36 @@
 
 Intel® P4 Studio Software Development Environment (SDE) is a set of packages for programming Intel’s line of programmable Ethernet Switches. The package also contains scripts for building and installing SDE. The fully automated script (`install.sh`) introduced in this release internally invokes the existing command line interface tool (`p4studio`) to ease the build and installation process. The following sections describe how to build, install, and run the Intel P4 Studio SDE.
 
+# What is included
+
+The following things are included in this repository:
+
++ Driver software
++ bfrt runtime API implementation
```

Comment:
The official name is BRI -- Barefoot Runtime Interface, which consists of two components (APIs):
  * The locally callable BfRt API (with bindings in C and C++)
  * The gRPC-based protocol, called BF Runtime, together with the server implementation in C++ and both C++ and Python client bindings (others can be generated using the provided proto files as well)
      ** The additional `bfrt_grpc.client` Python module provides easier-to-use, simplified interface to BF Runtime (used for most PTF tests, for example)